pkg/syncer/metadatasyncer.go

```diff
@@ -408,6 +408,30 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 		}()
 	}
 
+	// Trigger get pv to backingDiskObjectId mapping on vanilla cluster
+	pvToBackingDiskObjectIdFSSEnabled := metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.PVtoBackingDiskObjectIdMapping)
+	if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorVanilla && pvToBackingDiskObjectIdFSSEnabled {
+		pvToBackingDiskObjectIdMappingTicker := time.NewTicker(time.Duration(getVolumeHealthIntervalInMin(ctx)) * time.Minute)
```

Comment:
getVolumeHealthIntervalInMin?
This feature should have its own config values.

Reply:
Add a getPVtoBackingDiskObjectIdMappingIntervalInMin() similar to getVolumeHealthIntervalInMin. Add a new env variable accordingly.
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v2.5.0-rc.1/pkg/syncer/metadatasyncer.go#L109

Reply:
Added new api for pv to backingdiskobjectid feature.